### PR TITLE
[4.1] Load results on suggestion selection

### PR DESCRIPTION
**Pull request by sakiss**
Pull Request for Issue # .

### Summary of Changes
The current workflow with the smart search suggestions is to 1st select the suggestion and then submit the form (pressing enter or clicking on the submit button).

This simplifies the process and loads the results once a suggestion is selected.

### Testing Instructions
After applying the patch, run `npm install`
Use the smart search with the suggestions enabled and click on a suggestion to get the results straight.

**DO NOTE: That feature will not work on purpose, in the advanced form, for obvious reasons**

### Actual result BEFORE applying this Pull Request
1st select the suggestion and then submit the form

### Expected result AFTER applying this Pull Request
Get the results on selecting a suggestion.

### Documentation Changes Required
No

### CODE REVIEW

File: build/media_source/com_finder/js/finder.es6.js [hunk 1]
```diff
@@ -52,6 +52,14 @@
     }
   };
 
+  // Submits the form programmatically
+  const submitForm = event => {
+    const form = event.target.closest('form');
+    if(form) {
```
Comment (richard67) [inline build/media_source/com_finder/js/finder.es6.js]:
```suggestion
  const submitForm = (event) => {
    const form = event.target.closest('form');
    if (form) {
```
Fix code style errors reported by drone

**Comment by sakiss** [comment]
@richard67 Do you think that this fits in the J 4.0 branch?

**Comment by richard67** [comment]
> @richard67 Do you think that this fits in the J 4.0 branch?

I don't know, and it's not on me to decide since I'm not a release lead.

**Comment by sakiss** [comment]
I am closing that as i created a PR for J4.0. Let's how this will go.

**Pull request closed without merging**